### PR TITLE
[testnetv4-dev] Fix scilla runner output parsing issue

### PR DESCRIPTION
**Pull request by kaikawaliu**
### Implementation

- [x] **ready for review**

### Integration Test (Core Team)

- [ ] local machine test

- [ ] small-scale cloud test

### CODE REVIEW

File: src/libData/AccountData/AccountStoreSC.tpp [hunk 1]
```diff
@@ -466,18 +463,22 @@ bool AccountStoreSC<MAP>::ParseCreateContractOutput(
       return false;
     }
   } else {
-    std::string outStr{std::istreambuf_iterator<char>(in),
-                       std::istreambuf_iterator<char>()};
+    std::string outStr = {std::istreambuf_iterator<char>(in),
```
Comment (Gnnng) [inline src/libData/AccountData/AccountStoreSC.tpp]: This seems not fixing the issue. A new `outStr` is still being defined and masks the outter one.

Comment (iantanwx) [reply src/libData/AccountData/AccountStoreSC.tpp]: Can confirm that this causes failure. I think haichuan has fixed this in some other branch.

Comment (kaikawaliu) [reply src/libData/AccountData/AccountStoreSC.tpp]: Sorry it’s a miss. Already fixed

**Comment by iantanwx** [comment]
This works. Thanks @kaikawaliu !

**Pull request merged**